### DISCUSSION TITLE
Upgrade Vue2 to Vue3 in Symfony 5.4

### DISCUSSION THREAD

**Discussion by avinash-addweb** [symfony/symfony#51657] [Q&A]
We have an existing system built on Symphony 5.4 and frontend is Vue2. Now we are in the process of upgrading the Vue version to the latest which is Vue 3.

As per the process to upgrade the Vue, I have completed all required steps to migrate from Vue2 to Vue3.

Now when we load an application, not a Simple Due component is being loaded on the application. I have changed the Vue Loader configuration in the webpack.config.js file.

![Screenshot from 2023-09-14 19-45-38](https://github.com/symfony/symfony/assets/111571158/33956e6a-f1b0-4f52-98a2-eac081befb60)

I need help with a couple of questions as below:  

1. Is there anything specific I am missing for Symphony 5.4 and Vue 3? 
2. Symphony 5.4 and Vue 3 are supported together?

**Comment by stof** [comment]
Symfony itself does not care about which JS framework you use.

The code you put in a screenshot corresponds to the [webpack-encore](https://github.com/symfony/webpack-encore) package, not to Symfony itself, so it is not related to which version of Symfony you use.
